requirements.txt

```diff
@@ -1,3 +1,4 @@
 datajoint>=0.13.0
 element-interface>=0.4.0
+compress-multiphoton
```

Comment:
Which repo does this refer to?  Could find this on PyPI.

Reply:
It's this one that needs to be published: https://github.com/datajoint/compress-multiphoton

Reply:
Please install from source in the meantime.